cwe_api/Dockerfile

```
@@ -24,6 +25,13 @@ RUN if [ ! -d modelo_cwe ]; then \
       || { echo "Failed to download or unzip model"; exit 1; } \
     fi
 
+RUN if [ ! -d utils ]; then \
```

Hadolint

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)